ivy/functional/frontends/tensorflow/linalg.py

```diff
@@ -111,3 +111,8 @@ def l2_normalize(x, axis=None, epsilon=1e-12, name=None):
     square_sum = ivy.sum(ivy.square(x), axis=axis, keepdims=True)
     x_inv_norm = ivy.reciprocal(ivy.sqrt(ivy.maximum(square_sum, epsilon)))
     return ivy.multiply(x, x_inv_norm)
+
+
+@to_ivy_arrays_and_back
+def sqrtm(input, name=None):
+    return ivy.sqrtm(input)
```

Comment:
The reason for the failure is that `ivy.sqrtm` has not been implemented.

Please the [frontend api contributor guide](https://lets-unify.ai/ivy/contributing/open_tasks.html#frontend-apis) for options on how to proceed. Basically, you can try and implement sqrtm in the frontend as a composition of ivy functions, or if that's too tricky, you can raise an issue requesting that `sqrtm` be implemented in the ivy backends and work on another function.